pytest-embedded-wokwi/pyproject.toml

```diff
@@ -35,8 +35,8 @@ requires-python = ">=3.7"
 dependencies = [
     "pytest-embedded~=1.17.0a2",
     "toml~=0.10.2",
-    # Temporary workaround for Wokwi client - will be redirected to the official repo
-    "wokwi-client @ git+https://github.com/JakubAndrysek/wokwi-python-client.git@add-sync-client",
+    # Temporary workaround - will be fixed for stable release
+    "wokwi-client",
```

Comment:
Hi @JakubAndrysek I saw the https://github.com/wokwi/wokwi-python-client/releases/tag/v0.1.0 has been released. could you update the requirements to

`wokwi-client>=0.1.0,<1` here?